aea/crypto/cosmos.py

```diff
@@ -636,26 +692,37 @@ def try_execute_wasm_transaction(
                 f.write(json.dumps(tx_signed))
 
             command = [
-                "wasmcli",
+                self.cli_command,
                 "tx",
                 "broadcast",
                 os.path.join(tmpdirname, signed_tx_filename),
             ]
 
-            stdout, _ = subprocess.Popen(  # nosec
-                command, stdout=subprocess.PIPE, stderr=subprocess.STDOUT
-            ).communicate()
+            tx_digest_json = json.loads(self._execute_shell_command(command))
 
-        return stdout.decode("ascii")
+        hash_ = cast(str, tx_digest_json["txhash"])
+        return hash_
+
+    def execute_contract_query(
+        self, contract_address: Address, query_msg: JSONLike
+    ) -> Optional[JSONLike]:
+        """
+        Execute a CosmWasm QueryMsg. QueryMsg doesn't require signing.
+
+        :param contract_address: the address of the smart contract.
+        :param query_msg: QueryMsg in JSON format.
+        :return: the message receipt
+        """
+        result = self._try_execute_wasm_query(contract_address, query_msg)
+        return result
```

Comment:
what is the reason to create a method that just calls another one?
for compatibility with some interface?